examples/rl/md/rlhf_dummy_demo.md

```diff
@@ -0,0 +1,234 @@
+# Reinforcement Learning from Human Feedback (RLHF) - Dummy Demo Guide
+
+This guide explains the concept of Reinforcement Learning from Human Feedback (RLHF) and walks through the components of the accompanying dummy demo script `rlhf_dummy_demo.py`.
+
+## 1. What is Reinforcement Learning from Human Feedback (RLHF)?
```

Comment:
Since it is not really human feedback, but another model - should we call thsi RLAIF?